torch/distributed/elastic/utils/data/elastic_distributed_sampler.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import math
+
+import torch
+from torch.utils.data.distributed import DistributedSampler
+
+
+class ElasticDistributedSampler(DistributedSampler):
```

Comment:
Do we plan to consolidate all distributed samplers into the same folder (torch.utils or torch.distributed)?

Reply:
yea we should, it'll be done after the upstream is done and when we start "blending" the torchelastic modules to torch. The same needs to be done to `multiprocessing`, `logging`, `events`, `utils`.